vllm/config/scheduler.py

```diff
@@ -37,6 +38,11 @@ class SchedulerConfig:
     This config has no static default. If left unspecified by the user, it will
     be set in `EngineArgs.create_engine_config` based on the usage context."""
 
+    prefill_max_num_batched_tokens: int = None  # type: ignore
```

Comment:
Is mypy the reason that `type: ignore` is needed?

```suggestion
    prefill_max_num_batched_tokens: int = None  # type: ignore
```

Reply:
I have already used `Field(default=None)` for `enable_hybrid_chunked_prefill` and `prefill_max_num_batched_tokens` in the latest commit,, and removed `# type: ignore` in the .